src/openrct2/windows/_legacy.c

```diff
@@ -76,6 +77,9 @@ const rct_object_entry DefaultSelectedObjects[26] = {
         { 0x00000087, { "SCGWATER" }, 0 }       // Water Feature Theming
 };
 
+// rct2: 0x00F64F2C
+sint16 window_scenery_tab_entries[20][SCENERY_ENTRIES_BY_TAB + 1];
```

Comment:
This introduces hard coded number for no good reason. At the very least write this as `MAX_SCENERY_GROUP_OBJECTS + 1` (which is already much of a stretch AFAIC).

Reply:
I can't use `MAX_SCENERY_GROUP_OBJECTS` in the header, because that causes a circular dependency issue.